README.md

```diff
@@ -43,13 +47,15 @@ GRANT ALL PRIVILEGES ON `exercism_v3_test-0`.* TO 'exercism_v3'@'localhost';
 GRANT ALL PRIVILEGES ON `exercism_v3_dj_test-0`.* TO 'exercism_v3'@'localhost';
 ```
 
-### Running the server
+### Running the local servers
+
+We have a Procfile which executes the various commands need to run Exercism locally.
+On MacOSX we recommend using `hivemind` to manage this, which can be installed via `brew install hivemind`.
 
-Run both a webpack server and a Rails server
+To get everything started you can then run:
 
 ```bash
-bundle exec bin/webpack-dev-server 
-bundle exec bin/rails server
```

Comment:
Maybe still keep this if people don't have / can't have hivemind :)

Reply:
Yeah. I might add a bit that says "read the Procfile.dev and do the steps manually".